dev/doc/release-process.md

```diff
@@ -2,6 +2,11 @@
 
 ## As soon as the previous version branched off master ##
 
+In principle, these steps should be undertaken by the RM of the next
+release. Unfortunately, we have not yet been able to nominate RMs
+early enough in the process for this person to be known at that point
+in time.
```

Comment:
I am not sure about this; shouldn't we add instead the nomination process as a step / pre-requisite?

Reply:
Yep!